Deeploy/Targets/Neureka/Parsers.py

```
@@ -50,14 +51,18 @@ def parseNodeCtxt(self,
         #         and enforcing that the channels_first is false
         data_in = newCtxt.lookup(self.operatorRepresentation['data_in'])
         data_out = newCtxt.lookup(self.operatorRepresentation['data_out'])
-        weight = newCtxt.lookup(self.operatorRepresentation['weight'])
+        # MARCHIOA: weight depends on the type of convolution so it requires to be parsed by the child parsers
+        #           - PW -> 3-dim
+        #           - DW -> 4-dim
+        #           - Dense -> 4-dim
+        # weight = newCtxt.lookup(self.operatorRepresentation['weight'])
 
         if not all([
                 channels_first == False,
```

Ruff (0.16.0)

[error] Avoid equality comparisons to `False`; use `not channels_first:` for false checks

Replace with `not channels_first`

(E712)